radio/src/opentx.cpp

```diff
@@ -1102,7 +1104,7 @@ void getADC()
 
 #if defined(FLYSKY_HALL_STICKS)
     if (x < 4)
-      v = get_hall_adc_value(x) >> (1 - ANALOG_SCALE);
+      v = get_flysky_hall_adc_value(x) >> (1 - ANALOG_SCALE);
```

Comment:
Any way you could hide this behind the normal function?

Reply:
I am not sure where should I hide, this code is currently comes from NV14, I just try to refactorize the driver to horus, I think NV14 firmware should work on this change as well and finally we can have NV14 target removed.

Reply:
Oh! I see now! Sorry for not checking better earlier! Ok let's keep it like that, we can make it better later.